sections/telemetry_system.tex

```diff
@@ -395,12 +442,36 @@ \subsection{Data Transmission}
 
 In order to transmit telemetry data over radio, \gls{cuinspace} has developed a radio packet encoding format. This
 format can be found in Appendix \ref{apx:comm-spec-repos}. The format covers multiple different data types that are
-recorded about the rocket state, such as altitude, acceleration, \gls{gps} coordinates, etc. \cite{radio-comms}.
+recorded about the rocket state like altitude, acceleration, or \gls{gps} coordinates. \cite{radio-comms}
 Importantly, each radio packet starts with a header containing the amateur radio call-sign of the \gls{cuinspace}
 operator responsible for flying the system. \cite{radio-comms} The original specification did not include sufficient
 space to append a call-zone indicator to the call-sign, which has since been amended in the latest version. This is a
 requirement for Canadians broadcasting in the United States \cite{foreign-broadcast}, as \gls{cuinspace} does at
-\gls{sac}.
+\gls{sac}. \gls{cuinspace} plans on using this same packet encoding in its data logging for consistency and compatability
+with existing tools.
+
+The radio packet encoding format also had changes this year to increase the amount of useful data sent over the radio by
+cutting out as much duplicated or unnecessary information from the packet header and individual measurements (which are
+stored in "blocks" in the packet) as possible. These changes are listed below.
+
+\start{itemize}
+    \item Removal of a version field for packet headers, as the team has not supported older versions of the
+          specification in the past
+    \item Updating the packet length field in the packet header to count the number of blocks instead of number of bytes
```

Comment:
I think we could mention here that block header length will be fixed and included in the block header will be it's type, from it's type we can derive it's length on the backend.